client/engine/messageservice/test-messageservice.go

```diff
@@ -31,8 +31,8 @@ func NewTestMessageService(address types.Address) TestMessageService {
 	tms := TestMessageService{
 		address: address,
 		toPeers: make(map[types.Address]chan<- string),
-		in:      make(chan protocols.Message),
-		out:     make(chan protocols.Message),
+		in:      make(chan protocols.Message, 5),
+		out:     make(chan protocols.Message, 5),
```

Comment:
I've read somewhere or other [citation needed] that buffered channels are generally only a good practice in special circumstances and are a smell when applied in an unconsidered way.

Given the circumstance (a mock communications network) I don't think we need to worry too much about it.